src/editor/editorui.js

```diff
@@ -60,8 +54,92 @@ export default class EditorUI {
 		 */
 		this.focusTracker = new FocusTracker();
 
+		/**
+		 * Stores all editable elements used by the editor instance.
+		 *
+		 * @protected
+		 * @member {Array.<module:ui/editorui/editoruiview~EditorUIView>}
+		 */
+		this._editableElements = [];
+
+		/**
+		 * **Deprecated** since `v12.0.0`. This property is deprecated and should not be used. Use the property
+		 * from the subclass directly instead, for example
+		 * {@link module:editor-classic/classiceditorui~ClassicEditorUI#_view ClassicEditorUI#_view}.
+		 *
+		 * The main (top–most) view of the editor UI.
+		 *
+		 * @deprecated v12.0.0 This property is deprecated and should not be used. Use the property
+		 * from the subclass directly instead, for example
+		 * {@link module:editor-classic/classiceditorui~ClassicEditorUI#_view ClassicEditorUI#_view}.
```

Comment:
It is not a private property what is deprecated. It is the public getter. `@deprecated` + `@private` is a strange pair.

Reply:
Makes sense. I marked it that way because it was created only to deprecate `this.view`. Fixed it and added a comment reminding about this.